src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/TextServicesLoader.cs

```diff
@@ -12,17 +12,11 @@
 //
 //
 
-using System;
-using System.Runtime.InteropServices;
-using System.Security;
 using System.Threading;
-using MS.Internal;
 using Microsoft.Win32;
 using MS.Win32;
-using System.Diagnostics;
 
 #if WINDOWS_BASE
-    using MS.Internal.WindowsBase;
```

Comment:
nit: Similar to other changes, we can remove all the occurances of MS.Internal.xyz.